deepctr_torch/models/basemodel.py

```diff
@@ -424,12 +434,24 @@ def _get_loss_func(self, loss):
             loss_func = loss
         return loss_func
 
-    def _get_metrics(self, metrics):
+    def _log_loss(self, y_true, y_pred, eps=1e-7, normalize=True, sample_weight=None, labels=None):
+        # change eps to improve calculation accuracy
+        return log_loss(y_true,
+                        y_pred,
+                        eps,
+                        normalize,
+                        sample_weight,
+                        labels)
+
+    def _get_metrics(self, metrics, set_eps=False):
```

Comment:
Excellent! Solved the issue [55](https://github.com/shenweichen/DeepCTR-Torch/issues/55)